### ISSUE TITLE
Npm run dev / npm run build fail on execution. 

### ISSUE THREAD

**Issue by Sun-Wukong** [mcordero12/newark-thrives-test#1]
Hey Mo, I just tried to run both npm run commands. From what I can guess, the commands are valid, however, the files themselves may be missing. Attached are also the debug logs I've received.
[npm-debug.log](https://github.com/mcordero12/newark-thrives-test/files/1380455/npm-debug.log)

**Comment by mcordero12** [comment]
Hey Jason, what OS are you running this from?

**Comment by Sun-Wukong** [comment]
Afternoon Mo, heres my environment configuration:

* OS: Ubuntu 14.04 _via c9.io_
* npm version: 3.5
* nodejs version: v6.9.1

**Comment by mcordero12** [comment]
Do you have vue-cli installed? 

`npm install -g vue-cli`

**Comment by Sun-Wukong** [comment]
Quick update, I have attempted 3 clean standups, with the inclusion of installing vue-cli beforehand. I still receive the same error, and I noticed there are some missing folders, one that came to mind was named "dist", as it seems to be referenced in a few places. 

I know your code works on your machine, but may you try reproducing the same workflow from a clean point? I've followed the documented instructions and suggestions, but the error persists.
